src/main/java/mate/academy/rickandmorty/repo/CharacterRepository.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.rickandmorty.repo;
+
+import java.util.List;
+import mate.academy.rickandmorty.model.CharacterEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CharacterRepository extends JpaRepository<CharacterEntity, Long> {
+
+    CharacterEntity findByExternalId(String externalId);
+
+    List<CharacterEntity> findByNameContaining(String name);
+
```

Comment:
no reason for these blank lines
```suggestion
    CharacterEntity findByExternalId(String externalId);

    List<CharacterEntity> findByNameContaining(String name);
```